tests/src/main/scala/sttp/tapir/tests/package.scala

```diff
@@ -70,7 +69,21 @@ package object tests {
     .out(header[Int]("IntHeader") and stringBody)
 
   val in_json_out_json: Endpoint[FruitAmount, Unit, FruitAmount, Any] =
-    endpoint.post.in("api" / "echo").in(jsonBody[FruitAmount]).out(jsonBody[FruitAmount]).name("echo json")
+    endpoint.post.in("api" / "echo")
+      .in(jsonBody[FruitAmount])
+      .out(jsonBody[FruitAmount]).name("echo json")
+
+  val in_content_type_fixed_header: Endpoint[Unit, Unit, Unit, Any] =
+    endpoint.post.in("api" / "echo")
+      .in(header(Header.contentType(MediaType.ApplicationJson)))
+
+  implicit val mediaTypeCodec: Codec[List[String], MediaType, CodecFormat.TextPlain] = Codec.list(Codec.string).mapDecode {
```

Comment:
where do we need this?

Reply:
ah below I see :)

Reply:
I think defining a `Codec[String, MediaType]` should be enough, the list should then be automatically added